chrome/inject.browserify.js

```diff
@@ -77,7 +77,7 @@
   update(field(USERNAME_FIELDS), login.u);
 
   if (login.digits) {
-    alert(login.digits);
+    alert(login.label != "" ? login.label + ": " + login.digits : login.digits);
```

Comment:
How about this?

`alert((login.label || "OTP") + ": " + login.digits);`

Reply:
Makes sense to me - updated.